articles/libraries/lock/v10/customization.md

```diff
@@ -123,6 +124,16 @@ Examples of `allowedConnections`:
 
 ![Lock - Allowed Connections](/media/articles/libraries/lock/v10/customization/lock-allowedconnections-social.png)
 
+### allowShowPassword {Boolean}

```

Comment:
@jeffreylees In some other methods you also have screenshots of how the option looks, for example https://auth0.com/docs/libraries/lock/v10/customization#language-string- Since this one adds a checkbox, shouldn't we have a screenshots here too?